src/fragments/lib/analytics/flutter/record.mdx

```diff
@@ -28,27 +28,14 @@ However, it can take upwards of 30 minutes for the event to display in the Filte
 
 ## Flush events
 
-Events have default configuration to flush out to the network every 30 seconds. If you would like to change this, update `amplifyconfiguration.dart` with the value in milliseconds you would like for `autoFlushEventsInterval`. This configuration will flush events every 10 seconds:
-
-```json
-{
-  "UserAgent": "aws-amplify-cli/2.0",
-  "Version": "1.0",
-  "analytics": {
-    "plugins": {
-      "awsPinpointAnalyticsPlugin": {
-        "pinpointAnalytics": {
-          "appId": "AppID",
-          "region": "Region"
-        },
-        "pinpointTargeting": {
-          "region": "Region"
-        },
-        "autoFlushEventsInterval": 10
-      }
-    }
-  }
-}
+Events have default configuration to flush out to the network every 30 seconds. To modify the default value, assign the `autoFlushEventsInterval` property within the `options` parameter of the `AmplifyAnalyticsPinpoint()` constructor to your desired duration. This code snippet configures the plugin to flush events every 10 seconds.
```

Comment:
*nit:* this is a bit wordy. I think you trim it down since you have the example.

```suggestion
Events have default configuration to flush out to the network every 30 seconds. To modify the default value, assign the `autoFlushEventsInterval` property to your desired duration. The example below configures the plugin to flush events every 10 seconds.
```